www/src/pages/components/grid/react/index.mdx

```diff
@@ -68,6 +68,20 @@ import { ComponentHeader, ComponentFooter } from 'components/thumbprint-componen
 </Grid>
 ```
 
+### Centered grid
+
+Though this design pattern is often built using the Grid component, it is more easily solved by using Atomic classes: `mw7 center tc`.
```

Comment:
```suggestion
The `Grid` component should not be used for centered single-column grids. This can be built with [Atomic classes](https://thumbprint.design/atomic/): `mw7 center tc`.
```
The "often built using the Grid component" isn't really true since the React component doesn't support it.